bindings/nodejs/tests/types/block.spec.ts

```diff
@@ -11,28 +11,27 @@ import * as protocol_parameters_json from '../../../../sdk/tests/types/fixtures/
 import { Block, BlockId, parseBlock, ProtocolParameters } from '../../';
 
 describe('Block tests', () => {
-
     it('compares basic block tagged data payload from a fixture', async () => {
         const block = parseBlock(basic_block_tagged_data_payload_json.block);
         expect(block).toBeInstanceOf(Block);
         const params: ProtocolParameters = JSON.parse(JSON.stringify(protocol_parameters_json.params));
-        const expected_id = basic_block_tagged_data_payload_json.id as BlockId;
+        const expected_id: BlockId = basic_block_tagged_data_payload_json.id;
         expect(Block.id(block, params)).toEqual(expected_id);
     });
 
     it('compares basic block transaction payload from a fixture', async () => {
         const block = parseBlock(basic_block_transaction_payload_json.block);
         expect(block).toBeInstanceOf(Block);
         const params: ProtocolParameters = JSON.parse(JSON.stringify(protocol_parameters_json.params));
-        const expected_id = basic_block_transaction_payload_json.id as BlockId;
+        const expected_id: BlockId = basic_block_transaction_payload_json.id;
         expect(Block.id(block, params)).toEqual(expected_id);
     });
 
     it('compares validation block from a fixture', async () => {
         const block = parseBlock(validation_block_json.block);
         expect(block).toBeInstanceOf(Block);
         const params: ProtocolParameters = JSON.parse(JSON.stringify(protocol_parameters_json.params));
-        const expected_id = validation_block_json.id as BlockId;
+        const expected_id = validation_block_json.id;
```

Comment:
Curiosity, why is the explicit type not needed here?

Reply:
It's cause `expected_id` is a `string`, and `BlockId` is just an "alias" type for `string`, so it's the same (the next link that asserts doesn't complain cause the type matches). But i'll add it for consistency.